programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -386,6 +386,9 @@ pub mod gateway {
             Errors::SPLAtaAndMintAddressMismatch,
         );
 
+        let cost_gas = 5000; // default gas cost in lamports
+        let cost_ata_create = &mut 2039280; // default SPL ATA account creation rent fee in lamports
```

Comment:
Is it an universal constant? Any link we can provide how the value is determined? I would use a ceiling value to take some more security like `2040000` or `2100000`

Reply:
I think this number is only for ZRC20 `gas_limit` configuration reference. The `cost_ata_create` gets overwritten right after by `*cost_ata_create = bal_before - bal_after;` without being read.

I'm not against using rounded number because I myself also used it for Solana rent exampt [here](https://github.com/zeta-chain/node/blob/develop/pkg/constant/constant.go#L33)

Reply:
it should be a constant, but let me see if using a computed is viable.

Reply:
we'll use whatever the CPI call actually debits from the signer. No longer need this constant.

Reply:
addressed in a92a84f